src/components/RankingTable.js

```diff
@@ -10,25 +12,69 @@ const styles = {
 		width: '100%',
 		maxWidth: contentMaxWidth,
 		flexDirection: 'column',
-		backgroundColor: paper,
-		boxShadow: [[1, 1, 2, 2, shadow]],
-		border: [[1, 'solid', black]],
+		backgroundColor: yellow,
 		height: 'fit-content',
-		marginBottom: 16,
+		marginBottom: 50,
 	},
 	headerRow: {
-		textTransform: 'uppercase',
 		fontWeight: 'bold',
+		textAlign: 'left',
+		'& div': {
+			border: 0,
+		},
+	},
+	podiumRow: {
+		backgroundColor: black,
+		color: '#fff',
+		border: 'none',
+	},
+	hideLosers: {
+		display: 'none',
+	},
+	hideWinners: {
+		display: 'none',
+	},
+	starDivider: {
 		textAlign: 'center',
+		...robotoBold,
+		color: black,
+		lineHeight: '90%',
+		fontSize: 50,
 	},
 }
 
-const RenderRows = ({ players }) => (
+const RenderPodium = ({ players, classes }) => (
```

Comment:
This is a smart way of handling it, but thinking about being DRY (Don't repeat yourself) this is some repetitive code. See how this basically duplicates the `RenderRows` logic but adds a different class? Now we're mapping over the array of players twice which is fine with the small list for quadio, but might get unwieldy with HUGE lists of players. 

Check out [Array.prototype.slice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice). Think about how you might be able to use a single `RenderRows` component twice on lines 95 and 97.